tools/junitreport/pkg/parser/gotest/data_parser_test.go

```diff
@@ -21,12 +21,8 @@ func TestMarksTestBeginning(t *testing.T) {
 			testLine: "=== RUN 1234",
 		},
 		{
-			name:     "url",
```

Comment:
I don't know how to produce such output. AFAIK, there is always a function name. Am I wrong?